Horus widget model name

offer-shmuely opened the pull request:
![image](https://user-images.githubusercontent.com/7026911/57199882-05ce7880-6f8d-11e9-92b5-42a4a95362e0.png)

3djc committed:
Horus : do not display POT2 as 6P if it is not

bsongis committed:
More translations

bsongis committed:
More translations

bsongis committed:
More translations

bsongis committed:
CZ translations

bsongis committed:
Gyro added, dead code removed

bsongis committed:
CZ translations continued

bsongis committed:
Missing translation

bsongis committed:
Translations again

bsongis committed:
Assert added

bsongis committed:
Segfault removed

bsongis committed:
Compilation fix

bsongis committed:
Cosmetics

bsongis committed:
More work for X-Lite S/PRO

bsongis committed:
GyrX / GyrY eeprom support on X-Lite S/PRO

bsongis committed:
[X3] Only 5 switches

bsongis committed:
Renaming PXX / ACCESS modules constants

bsongis committed:
ACCESS started in Companion

kilrah committed:
Remove obsolete option

bsongis committed:
As X3 is X9LITE PCBX9 should be defined

3djc committed:
rename x3 to x9lite

3djc committed:
rename x3 to x9lite

bsongis committed:
Compilation fix

bsongis committed:
Compilation fix

bsongis committed:
Logical issue: X9DP is NOT an X9D. Will be fixed better when we introduce radio family definitions

bsongis committed:
Logical issue: X9DP is NOT an X9D. Will be fixed better when we introduce radio family definitions

3djc committed:
cosmetics

3djc committed:
cosmetics

bsongis committed:
New bunch of translations

bsongis committed:
[X3] missing MIXSRC line (+ an assert to avoid it happens again)

bsongis committed:
Renaming continued

bsongis committed:
Compilation fix

bsongis committed:
[X7] Bootloader fix

bsongis committed:
[X9 Lite] Simulator keys fixed

bsongis committed:
[X7] Simulator keys fixed

kilrah commented:
Similarly to the other issue, you are overstepping the zone boundary:

![capture_x12s_19-05-13_18-30-08](https://user-images.githubusercontent.com/6065069/57638538-40986800-75ae-11e9-9b83-fbdf188f3b3f.png)

offer-shmuely commented:
Is that a problem on the top bar?
There is no other option, since we can’t select different layouts of top bar.

How do you suggest to write (widget or not) the model name in big letters?

bsongis committed:
Update CREDITS.txt

kilrah committed:
VC++ compilation

bsongis committed:
3djc/x7 additional switches (#6422)

X7, add two additional switches

bsongis committed:
Remaining X3 in code

bsongis committed:
Bug with [EXIT Long] in TX / RX Options screen

bsongis committed:
Compilation fix

bsongis committed:
Cosmetics

bsongis committed:
Cosmetics

bsongis committed:
Cosmetics

bsongis committed:
[X10] Freeze issue fixed

bsongis committed:
Cosmetics

3djc committed:
Allow CROSSFIRE serial telemetry mirror

bsongis committed:
HORUS fixes

bsongis committed:
CZ next round of translations

bsongis committed:
HORUS fixes

3djc committed:
Horusgets 60 sensors, not 64

3djc committed:
Build simu for new radios

bsongis committed:
Compilation options updated

bsongis committed:
Compilation options updated

bsongis committed:
Compilation options updated

bsongis committed:
Cosmetics

3djc committed:
Verbose nighlies

kilrah committed:
Compilation fix

3djc committed:
Fix macosx compile

3djc committed:
Fix macosx compile

3djc committed:
Fix macosx compile

bsongis committed:
Compilation options updated

bsongis committed:
Windows compilation fight

3djc committed:
Allow accst protocols on xlites

bsongis committed:
Windows compilation fight

bsongis committed:
Revert "Allow accst protocols on xlites"

This reverts commit 5040e518

bsongis committed:
Revert "Compilation options updated"

This reverts commit 68a9edc8

bsongis committed:
Windows compilation fight

bsongis committed:
Windows compilation fight

bsongis committed:
Should work if we compile with -DWIN_DO_FULL_INSTALL=YES -DSIMU_TARGET=NO

bsongis committed:
License.txt to ISO-8859-1

bsongis committed:
Windows compilation fight

bsongis committed:
Windows compilation fight

bsongis committed:
Windows compilation fight clean

bsongis committed:
Windows compilation fight next round

bsongis committed:
Windows compilation fight next round

bsongis committed:
Windows compilation fight next round

bsongis committed:
Windows compilation fight next round clean

bsongis committed:
External module list depends on compilation options
- option PPM=NO is added for radios which don't support PPM anymore
- option SBUS=NO
- option DSM2=NO

bsongis committed:
Fixes in channels order in the Multi wizard

bsongis committed:
Options for building FrSky release firmware updated

bsongis committed:
Translations fix

bsongis committed:
Translations fix

bsongis committed:
Quel talent !

3djc committed:
Update firmware stamps asap

3djc committed:
small companion fixes

3djc committed:
Fix #•6425

3djc committed:
cosmetics

3djc committed:
Fix eeprom load multiples warnings

bsongis committed:
Untranslated strings splitted from others

3djc committed:
cosmetics

bsongis committed:
ACCST protocol support added

bsongis committed:
Strings big cleaning session (#6426)

Strings cleaning session

bsongis committed:
Companion new error detected on flash (when no device found)

bsongis committed:
Compilation fix

bsongis committed:
X10 extra switches sticks (#6423)

[X10] 2 extra sticks switches + 2 extra pots

bsongis committed:
String change

bsongis committed:
String change

bsongis committed:
String change

bsongis committed:
String change

bsongis committed:
Compilation fix

bsongis committed:
FourCC missing for new boards in Companion

bsongis committed:
Will raise a warning next time we forget a FourCC or an EEPROM size

bsongis committed:
Compilation fix

bsongis committed:
Compilation fix

bsongis committed:
Compilation fix

bsongis committed:
Compilation fix again (no possible to avoid it with VC++ ???)

bsongis committed:
VC++ warnings fixes

Commit:
Update gui_common.cpp

Added:
 - BAYANG DHD_D4
 - MJXQ PHOENIX and Freq tuning for E010
 - E01X E016H
 - GD00X V1 and V2
 - BUGSMINI BUGS3H

bsongis committed:
ACCST on ACCESS (#6430)

ACCST on ACCESS

ku2works committed:
updated japanese translation. (#6429)

3djc committed:
Fix ALT displayed in TopBar in FAI mode (2.3 version) (#6367)

SHezinger committed:
Some german translations added or corrected (#6344)

Some german translations added or corrected

Commit:
Wrong search & replace

bsongis committed:
Fix ported from #6331

bsongis committed:
#6331

3djc committed:
Prevent dist calculation overflow

3djc committed:
Improve nightly outputs

3djc committed:
Fix GPS overflow

3djc committed:
Make nightlies process more robust

3djc committed:
Make nightlies process more robust

kilrah committed:
[companion] Fix Bluetooth fields availability, add ownerId and gyro

3djc committed:
Make nightlies process more robust

kilrah committed:
Reserve gyro data for Horus

kilrah committed:
Compilation fix

3djc committed:
Multi compile fix

3djc committed:
日本語オプションを追加

3djc committed:
Add japaneese translation file

3djc committed:
Temporary fix for Sky9X compilation issue

3djc committed:
Use the Force, Luke

bsongis committed:
SKY9X-REVA removed as it doesn't even build since 2.2

bsongis committed:
Compilation fix

bsongis committed:
Unused file removed

bsongis committed:
Removing PPM support also removed Trainer / Trainee feature

3djc committed:
CROSSFIRE is no more a default, but firmware option (saves RAM)

3djc committed:
CROSSFIRE module availability is based on sport sharing

Commit:
Companion enhancements for checklist editor, model print and compare (2 3 branch) (#6434)

* Model print and compare include GFs enhancement

* Model print and compare include checklist contents

* Add missing brackets and code tidy up

* Print message if cannot open or read checklist file

* Missing GV label translation
Move some checklist code to helpers

* Refactor to use helpers and add cursor position

* Move improved version of removeAccents to helpers and apply to checklist

* Force fixed width font for edit window

bsongis committed:
Fixes #6439

bsongis committed:
[ACCESS] More bits for the RF protocol enum

bsongis committed:
Taranis X9D REV4a support removed (didn't compile since 2.2)

bsongis committed:
Problem in Failsafe HOLD / NONE strings

3djc committed:
Allow PXX1 only

3djc committed:
Fix wrong BT password
Thix fixes #6440

3djc committed:
REvert previous fix and do it differently

Commit:
Fixes missing or invalid SD path (#6441)

bsongis committed:
XLite EEPROM 218 import fix

3djc committed:
Improve massively splash screen ! Thx MHotar !!!

3djc committed:
Fix fiel permissions

bsongis committed:
CMake options printed when building a firmware

bsongis committed:
Companion import fix

bsongis committed:
O9X FourCC support removed

3djc committed:
Fix companion import
This fixes #6443

bsongis committed:
Fix for negative gvars precision - inspired by #6446 thanks @ elecpower

Commit:
Add mouseover tool tips to label fields with context menus (#6448)

bsongis committed:
Aie

kilrah committed:
Fix X10 analogs

bsongis committed:
Merge bitfield macros

3djc committed:
Fix PPMUS

3djc committed:
Add Jumper t12 support by 3djc (#6451)

* Jumper T12 - companion part

* more work

* Add definitions

* fix libsimulator key handling

* Fix simu keys handling

* Cosmetics

* Continued

* Fix wrong regplace

* Fix wrong regplace

* Please travis

* Please travis

* Add T12 to Travis
Switch naming and display

* Cosmetics

* Work on modules

* Work on modules

* typo

* Fix navigation

* Cosmetics

* Introduce NAVIGATION_type

* Fix

* Make NAVIGATION_type more future proof

* Lots of cleanups and improvements

* Cosmetics

* Fix internal module menu

* Cosmetics

* Fix screen reversed

* Small fixes

* Fix navigation issue

* Please travis

* Fix LCD contrast

* Fix T12 switch layout

* Fix T12 keys screen and all radios keys translations

* Complete rework of SWITCHES screen for all radios

* comsetics

* VC++ compilation

3djc committed:
Prep for switch refactoring

3djc committed:
Typo

3djc committed:
Fix missing

3djc committed:
Switch refactoring continued

3djc committed:
Fix duplicate

bsongis committed:
Too long string on 128x64 screens

kilrah committed:
Properly disable BT when module not present

(cherry picked from commit 2bc9a00f32a29a085e4764042ffc7d61019944f1)

3djc committed:
X12S has Gimbal switchs handling as well

bsongis committed:
Bsongis/power control chip (#6445)

FRSK files support added

bsongis committed:
Now module / receiver / sensor firmware update from .frsk files should be OK

bsongis committed:
Compilation fix

3djc committed:
Fix X7 switches on main view

3djc committed:
Switches refactoring (#6454)

Switches refactoring

bsongis committed:
CZ translations

bsongis committed:
Bsongis/accst support over pxx2 (#6455)

ACCST support over PXX2

bsongis committed:
Refactoring

bsongis committed:
Cosmetics

bsongis committed:
Compilation fix

bsongis committed:
Compilation fix

bsongis committed:
Unused code removal

bsongis committed:
Compilation fix

bsongis committed:
X9 Lite powering issue

bsongis committed:
warning removed

bsongis committed:
R9M ACCESS / ACCST refactoring

bsongis committed:
Update build options

3djc committed:
Introduce AUX_SERIAL (#6456)

bsongis committed:
[XLiteS / X9 Lite] Will only propose "ACCESS / ACCST D16 / ACCST LR12" (no ISRM)

LapinFou committed:
Initial version

bsongis committed:
Module names constants / strings changed again

LapinFou committed:
Updated Companion French translation for OpenTX 2.3

bsongis committed:
R9M bugfix

bsongis committed:
R9M improvements

3djc committed:
Clean outdated and unused pxx2 code (#6458)

Remove outdated PXX2 code from X10 / X12S

bsongis committed:
Bsongis/menus refactoring (#6463)

Menus refactoring

bsongis committed:
Update CREDITS.txt

bsongis committed:
R9M warning (#6464)

R9M warning + refactoring

bsongis committed:
String change

MRC3742 committed:
Multi protocol updates to MPM version 1.2.1.51 (#6466)

MRC3742 committed:
Travis build error fix for PR #6466 Update MPM for 2.3

3djc committed:
Remove non working Theme

bsongis committed:
Update CREDITS.txt

bsongis committed:
Negative test inversion. Internal module mode displayed wrongly on X9 Lite

bsongis committed:
ACCST D16 / ACCESS mode removed until it is ready

bsongis committed:
Same shutdown animation on all boards

bsongis committed:
ACCST D16 is OK on ACCESS modules now

bsongis committed:
Fixes #6459

bsongis committed:
Switches position was not optimal in Main view [Switches]

kilrah committed:
Xlite has no trainer port

Commit:
Fixes 5602 Special Functions Inc/Dec Gvar (#6442)

Fixes #5602 (2.3)

bsongis committed:
Cosmetics

shellixyz committed:
Add vario sensor to CRSF telemetry

3djc committed:
T12 to use 9x navigation + navigation refactoring  (#6474)

* Move T12 to more suited 9x navigation

* Navigation refactoring

* Navigation refactoring continued

* Channels count in ACCESS mode fix (24 channels max)

bsongis committed:
Cosmetics

bsongis committed:
Radio / Version: FCC / EU variant displayed

bsongis committed:
BIND in ACCST mode didn't take care of 8CH / 16CH

bsongis committed:
Cosmetics

3djc committed:
Remove 1ms for mixer to pulse XJT (now 2ms)

bsongis committed:
Refactoring around trainer

bsongis committed:
ACCESS module type added to Companion

bsongis committed:
Compilation fix

bsongis committed:
Compilation fix

bsongis committed:
Cosmetics

3djc committed:
Fix critical issue when flashing 2.2.x firm from 2.3 bootloader (making 2.3 BL init wrong interupt not handled by 2.2 firm)

bsongis committed:
Cosmetics

bsongis committed:
Rename serial2 to aux_serial

bsongis committed:
Compilation fix

bsongis committed:
Compilation fix

bsongis committed:
ACCESS support started in Companion

bsongis committed:
Renaming "Power control chip" to "Power management unit"

bsongis committed:
Fix switch warning for X9 Lite

bsongis committed:
Always use 230400bauds for PXX2 on external module

3djc committed:
Improve switch layout display

3djc committed:
Companion T12 support (#6480)

* Introduce T12 own variant, radio side

* T12 Companion support continued

* Remove warnings

* Cosmetics

3djc committed:
Please Travis

bsongis committed:
Cosmetics

bsongis committed:
Fix an issue in statistics display

bsongis committed:
Cosmetics

bsongis committed:
Update CREDITS.txt

bsongis committed:
R9M subtype selection fix

bsongis committed:
Switches fix (because we have more than 256 switches today)

bsongis committed:
CZ translations

bsongis committed:
X7 didn't have the S.PORT connector enabled

3djc committed:
Fix nightlies compilation and add multi to Travis (it is mainstream after all)

bsongis committed:
S.PORT Flash fix

bsongis committed:
Lua warning popups could not be canceled!

bsongis committed:
Bsongis/radio tools (#6484)

Lua tools now displayed in RADIO / TOOLS
Add a function to write CENTERED text with lcdDrawText

bsongis committed:
PXX synchro using Hearbeat (#6482)

* [PXX] Latency using heartbeat sync optimized for X7 (stable @ 12ms)
* [PXX] Latency using heartbeat sync optimized for XLite Pro (stable @ 9ms)

Commit:
Add RX numbers to CPN models list (2.3) (#6447)

Add RX numbers to models list (#5088)

bsongis committed:
Typo

Commit:
Merge remote-tracking branch 'origin/2.3' into 2.3

Commit:
Merge branch '2.3' into Horus_widget_model_name
